play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -480,18 +491,21 @@ Starting Wealth: $1700+100*Luck + 2000
 Class Feats:
 ----------------------
 Pin Target
-Select a target. +25% to hit. On hit, target is Pinned. If the target attempts 
-to move from their pinned cover, you may roll a DC 60 sniping skill check to 
-hit them. For the remainder of the encounter, you get a +10% to hit the 
-selected target. If your rifle has Pin listed as an attribute, then any time 
-the selected target is pinned you may attack them reflexively if they move 
-from their pinned cover. If the target dies you may select a new target.
+If you fire at a target that is in cover, you can choose to "Pin" the target. Only 1 target may be
+Pinned by the marskman at a time. If the target attempts to move from cover within a 180 degree field
+of view infront of you, you may roll a DC 60 sniping skill check to hit them, as a free action. For the
+remainder of the encounter, you have a +10% to hit the pinned target with a firing action. If the target moves, 
+it is still considered pinned, and if the target dies or escapes from the battlefield you may pin a new target.
+Pin can only be activated by a pinned target once per Marksman player's turn. Only usable with long-rifles.
 |
 |
 |
 V
 Multi-Pin
-You may pin up to PER targets at a time. +20% to hit a Pinned target.
+You can now Pin a maximum of PER/2 (round down) targets at  time (you still have to fire at a 
+target to pin it). +20% to hit a Pinned target (instead of +10%). The Multi-Pin ability can be 
+activated by a pinned target once, per target, per Marksman player's turn. (I.e., 3 Pinned targets can 
```

Comment:
> The Multi-Pin ability can be activated by a pinned target once, per target, per Marksman player's turn.

I think we can write that more clearly:
" The Multi-Pin ability can be activated by a pinned target, once per target, per Marksman player's turn."
or "one activation per target" vs "once per target"
what do you think?